ArduCopter/Copter.h

```diff
@@ -567,7 +567,9 @@ class Copter : public AP_Vehicle {
     // Tradheli flags
     typedef struct {
         uint8_t dynamic_flight          : 1;    // 0   // true if we are moving at a significant speed (used to turn on/off leaky I terms)
+#if AP_INVERTED_FLIGHT_ENABLED
```

Comment:
does this show up in logs as a binary value? would this invalidate the docs?

Reply:
As in the other comment, I can't see anywhere this structure is externally visible, thus me not attempting to preserve the offsets of the subsequent bits.

Also, `coll_stk_low` in  this structure. is probably incorrectly specified?!